Tools/build/compute-changes.py

```diff
@@ -21,11 +21,27 @@
 GITHUB_DEFAULT_BRANCH = os.environ["GITHUB_DEFAULT_BRANCH"]
 GITHUB_CODEOWNERS_PATH = Path(".github/CODEOWNERS")
 GITHUB_WORKFLOWS_PATH = Path(".github/workflows")
+
 CONFIGURATION_FILE_NAMES = frozenset({
     ".pre-commit-config.yaml",
     ".ruff.toml",
     "mypy.ini",
 })
+UNIX_BUILD_SYSTEM_FILE_NAMES = frozenset({
+    Path("aclocal.m4"),
+    Path("config.guess"),
+    Path("config.sub"),
+    Path("configure"),
+    Path("configure.ac"),
+    Path("install-sh"),
+    Path("Makefile.pre.in"),
+    Path("Modules/makesetup"),
+    Path("Modules/Setup"),
+    Path("Modules/Setup.bootstrap.in"),
+    Path("Modules/Setup.stdlib.in"),
+    Path("Tools/build/regen-configure.sh"),
```

Comment:
cc @erlend-aasland -- are there any Unix (non-Windows) build files we've missed here?